hacks/count.js

```diff
@@ -1,3 +1,19 @@
+function delta(currentCount, previousCount) {
+    if (!mongo_hacker_config['count_deltas']) {
+        return "";
+    }
+    var delta = Number(currentCount - previousCount);
+    delta = isNaN(delta) ? colorize("(first count)", { color: 'blue' }) :
+        (delta == 0) ?  colorize("(=)", { color: 'blue' }) :
+            (delta > 0) ? colorize("(+" + delta.commify() + ")", { color: 'green' }) :
+                (delta < 0) ? colorize("(" + delta.commify() + ")", { color: 'red' }) :
+                    (delta + " not supported");
+    return " / " + delta.pad(21); // FIXME use "dynamic" padding
```

Comment:
> would like it if this was fixed before merging

I thought you might :smile: - the reason I shied away from it is that it requires a pretty significant refactor of the `printPaddedColumns()` function in `hacks/helpers/js`, mainly to support multiple columns _(instead of just two, which it does currently)_